pkg/customizations/oscap/oscap.go

```diff
@@ -40,6 +40,21 @@ const (
 	defaultRHEL9Datastream   string = "/usr/share/xml/scap/ssg/content/ssg-rhel9-ds.xml"
 )
 
+type RemediationConfig struct {
+	Datastream         string
+	ProfileID          string
+	TailoringPath      string
+	CompressionEnabled bool
+}
+
+type TailoringConfig struct {
+	RemediationConfig
+	Filepath   string
+	NewProfile string
```

Comment:
Nitpick: Is this the new profile ID? If so, would it make sense to call it `NewProfileID` or `TailoredProfileID`? :thinking:

Reply:
Yeah good suggestion